doc/sphinx/proof-engine/categories.txt

```diff
@@ -0,0 +1,58 @@
+A first stab at categories:
+
+Values
+  Types of values
+  Syntactic values
+  Tactics in terms
```

Comment:
Is there a reason this is so early?  Tactics in terms are sort of an aside, while substitution and let and application seem more central?

Reply:
How about in this subsection after "Function construction and application"?  That seems better to me than putting it under "Manipulating values".